src/github3/repos/repo.py

```diff
@@ -1524,6 +1524,27 @@ def ignore(self):
             notifications.RepositorySubscription, json
         )
 
+    @requires_auth
+    def unignore(self):
+        """Unignore notifications from this repository for the user.
+
+        .. versionadded:: 1.0
```

Comment:
It's not added in 1.0. It's added in 1.3 at this point

Reply:
Also the methods are pretty much alphabetical, so I think this needs to be nearer to `unsubscribe`